### ISSUE TITLE
Unable to load forem-theme-base gem

### ISSUE THREAD

**Issue by ghost** [rubysherpas/forem#699]
The repo doesn't exists

**Comment by sintro** [comment]
Yep. Sad, but this gem is dying (or already dead). Probably, this gem https://github.com/thredded/thredded is better (did not use it yet), so, it is not problem.

**Comment by ghost** [comment]
@sintro So i replace that gem fro thredded ?

**Comment by sintro** [comment]
I am talking about Forem gem. Read bold text in README of this repo.

**Comment by ghost** [comment]
ooohhh, so bad to hear that

**Comment by frankie-loves-jesus** [comment]
Anybody willing to upgrade forem to rails5?

**Comment by ghost** [comment]
@frankie-loves-jesus tought this is dead

**Comment by frankie-loves-jesus** [comment]
it doesnt have to be

**Comment by bodrovis** [comment]
Some years ago I implemented a bunch of updates and fixes but some of them are still not merged because the owners are having too much other things to do. And to be honest the gem (and the themes) have a handful of problems that needs to be taken care of.

**Comment by frankie-loves-jesus** [comment]
Would you be willing to become the new repo owner @bodrovis? :)

**Comment by bodrovis** [comment]
@frankie-loves-jesus Not sure, probably not now, moreover Ryan did not propose me anything. I was using Forem on one of my sites about 4 years ago but now I don't, so no real need for me to work on this gem. Though if someone else wishes to support the project then I might step in.
